python/paddle/nn/functional/activation.py

```diff
@@ -13,48 +13,17 @@
 # limitations under the License.
 
 # TODO: define activation functions of neural network
-from ...fluid.layers import brelu  #DEFINE_ALIAS
-# from ...fluid.layers import erf  #DEFINE_ALIAS
-from ...fluid.layers import maxout  #DEFINE_ALIAS
-# from ...fluid.layers import soft_relu  #DEFINE_ALIAS
-from ...fluid.layers import swish  #DEFINE_ALIAS
-from ...fluid.layers import sigmoid  #DEFINE_ALIAS
-from ...tensor.math import tanh  #DEFINE_ALIAS
-from ...tensor.math import tanh_  #DEFINE_ALIAS
+from ...fluid.layers import brelu  # noqa: F401
+# from ...fluid.layers import erf  # noqa: F401
```

Comment:
looks like the `noqa` comments are inserted by `sed`ing `#DEFINE_ALIAS`, this could be problematic, see comments above.